shoes-swt/lib/shoes/swt/text_block.rb

```diff
@@ -44,10 +44,11 @@ def contents_alignment(current_position)
       end
 
       def adjust_current_position(current_position)
-        current_position.y = @dsl.absolute_bottom
+        current_position.y = @dsl.absolute_bottom + 1
```

Comment:
In `Slot` we have a `NEXT_ELEMENT_OFFSET` constant and I think another one in Dimension, maybe we should reuse one of them or just have one here as well - makes it a bit more readable :)